### DISCUSSION TITLE
Please Add Support of Animated 3D Models

### DISCUSSION THREAD

**Discussion by iRealMMSGit** [Smithsonian/dpo-voyager#262] [Ideas]
I love the Voyager Authoring tool, and I wish this tool would support 3D .glb files that contain animations.

**Comment by gjcope** [comment]
Thanks! Yes, we have that on our roadmap. Since Voyager uses [Three.js](https://github.com/mrdoob/three.js), it already has support for animated .glb files. We just need a good UI/system to configure how and when you can play them. We've started a branch for that here: https://github.com/Smithsonian/dpo-voyager/tree/dev-actions but would definitely appreciate any ideas/contributions!

**Comment by gjcope** [comment]
The v0.53.0 release now includes the Action Manager to create triggers for top-level animations in glb files. If you find any issues or have any ideas for future functionality on this please let us know!
